update README.md to include new power components for v0.8 release

rsnancollas opened the pull request:
Updates power components for v0.8 release.

rsnancollas committed:
update README.md to include new power components for v0.8 release

Carrotman42 commented on support-docs/README.md:
```diff
@@ -244,6 +244,220 @@ The table below gives a description of the properties used to specify a switchin
+|batteryConfig|battery configuration supported by the device|array of String| |
+|batteryCellChemistry|battery cell chemistry supported by the device|array of String| |
+|inputPowerSource|input power source supported by the device|array of String| |
+|inputCurrentAccuracy|accuracy of input current when set|unit| |
```
It looks like you changed up the script to strip out the "definitions.json#/" prefix (https://github.com/edatasheets/digital-datasheets/blob/main/support-docs/README.md#3171-mosfet). I think we should be consistent. I'm assuming you prefer to strip the prefix; it is admittedly verbose, so I could get behind it. I just want to make sure we were on the same page, and also so that we can decide to update the cases that have already been committed.

rsnancollas replied:
Oh that's a mistake. I changed that for some other debug. Let me get the original version and fix that :)

Carrotman42 reviewed (changes requested):
In addition to the inline comment I left, please update the CL description to indicate how you generated this information. (i.e., you used the script, presumably)

Carrotman42 reviewed (approved):
Otherwise LGTM

Carrotman42 commented on support-docs/README.md:
```diff
@@ -244,6 +244,220 @@ The table below gives a description of the properties used to specify a switchin
 |package|Package definition of the switching regulator|Object| |
 |complianceList|List of compliances met by the switching regulator component|Array| |
 
+####  3.10.2	 Ldo
```
The above section is already `3.10.2`, this should be `3.10.3`. The subsequent sections you added also need to be changed.

rsnancollas replied:
Good call - fixed this.

rsnancollas committed:
updated README for power components

Used a version of readme autogeneration script to generate new readme for
battery charger and backlight driver and fixed numbering in README

rsnancollas committed:
Fixed numbering of Power Components README
